src/util/opening_hours.cpp

```diff
@@ -218,9 +218,9 @@ struct opening_hours_grammar : qi::grammar<Iterator, Skipper, std::vector<Openin
         weekday_sequence = (weekday_range % ',')[ph::bind(&OpeningHours::weekdays, _r1) = _1];
 
         weekday_range
-            = wday[_a = _1, _b = _1]
-            >> -(('-' >> wday[_b = _1])
-                 | ('[' >> (nth_entry % ',') >> ']' >> -day_offset))
+            = (wday[_a = _1, _b = _1]
```

Comment:
@oxidase thank you for this explanation. This makes a lot more sense to me now.
I believe simply having a bit more context on what kind of format the grammar parses might already help here (`Parsing Mo-Fr into ph::construct<>(1,5)` or `Sa` into `ph:construct<6,6>`).  I agree that the longer explanation here might be a bit much, but it is already accessible now via git-blame, so I guess we are golden here.